deb-get

```diff
@@ -100,6 +100,10 @@ function elevate_privs() {
     fi
 }
 
+function pull_url_redirect() {
```

Comment:
This is not used. Please remove it or suggest it in a separate PR, explaining its purpose.

Reply:
the pull_url_redirect function was originally written to be used for getting the master pdf version info - but started to get too complicated so I just left it in the deb_master-pdf-editor function